src/TicTac.java

```diff
@@ -13,30 +14,51 @@ public TicTac() {
                 this.squares[i][j] = '.';
             }
         }
+        this.turn = 'O';
```

Comment:
you could realistically do something like `Color` or `PieceKind` where you make an enum with `X` and `O` values. probably overkill for a tiny project like this but if it was anything real-world I'd be more adamant about it